src/UI/Card/Card.tsx

```diff
@@ -2,19 +2,12 @@ import React from "react";
 import classes from "./Card.module.scss";
 
 type CardProps = {
-    className: string;
-    children: React.ReactNode;
+  className: string;
+  children: React.ReactNode;
```

Comment:
los props idealmente siempre dejarlos opcionales
```suggestion
  className?: string;
  children?: React.ReactNode;
```